hermes/serializers.py

```diff
@@ -0,0 +1,18 @@
+from hermes.models import Message
+from rest_framework import serializers
+
+
+class MessageSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
Do we need this serializer at all at the moment?

Reply:
the django rest framework API view is currently broken. I think we probably want that fixed in the long run, no?